sys-apps/fwupd/fwupd-1.3.2.ebuild

```diff
@@ -52,69 +68,68 @@ RDEPEND="${PYTHON_DEPS}
 	pkcs7? ( >=net-libs/gnutls-3.4.4.1:= )
 	redfish? ( sys-libs/efivar )
 	systemd? ( >=sys-apps/systemd-211 )
-	thunderbolt? ( sys-apps/thunderbolt-software-user-space )
+	thunderbolt? (
+		sys-apps/thunderbolt-software-user-space
+	)
 	uefi? (
+		app-crypt/tpm2-tss
 		media-libs/fontconfig
 		media-libs/freetype
 		sys-boot/gnu-efi
+		sys-boot/efibootmgr
 		>=sys-libs/efivar-33
 		x11-libs/cairo
 	)
 "
-DEPEND="${RDEPEND}
-	$(vala_depend)
-	x11-libs/pango[introspection]
-	nvme? (	>=sys-kernel/linux-headers-4.4 )
-	test? ( net-libs/gnutls[tools] )
-"
-BDEPEND="
-	>=dev-util/meson-0.47.0
-	virtual/pkgconfig
-	doc? ( dev-util/gtk-doc )
-	man? (
-		app-text/docbook-sgml-utils
-		sys-apps/help2man
-	)
-"
+RDEPEND="${DEPEND}"
 
 # required for fwupd daemon to run.
 # NOT a build time dependency. The build system does not check for dbus.
 PDEPEND="sys-apps/dbus"
 
-PATCHES=(
-	"${FILESDIR}/${P}-without_systemd_build_fix.patch"
-)
+pkg_setup() {
+	if use nvme; then
+		kernel_is -ge 4 4 || die "NVMe support requires kernel >= 4.4"
+	fi
+}
 
 src_prepare() {
 	default
 	sed -e "s/'--create'/'--absolute-name', '--create'/" \
 		-i data/tests/builder/meson.build || die
 	sed -e "/'-Werror',/d" \
 		-i plugins/uefi/efi/meson.build || die
+	# c.f. https://github.com/fwupd/fwupd/issues/1414
```

Comment:
(if this was my package, I'd prefer a patch instead of multiple sed lines already)

Reply:
Going to defer to @Polynomial-C

Reply:
`sed` is fine but we should check occasionally on new version bumps if it is still needed.

Reply:
Checked, both of the top ones are no longer necessary